dev/benchmarks/platform_views_layout/test_driver/scroll_perf_ad_banners_test.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart' hide TypeMatcher, isInstanceOf;
+
+void main() {
+  group('scrolling performance test', () {
+    late FlutterDriver driver;
+
+    setUpAll(() async {
+      driver = await FlutterDriver.connect();
+
+      await driver.waitUntilFirstFrameRasterized();
+    });
+
+    tearDownAll(() async {
+      driver.close();
+    });
+
+    Future<void> testScrollPerf(String listKey, String summaryName) async {
+      // The slight initial delay avoids starting the timing during a
+      // period of increased load on the device. Without this delay, the
+      // benchmark has greater noise.
+      // See: https://github.com/flutter/flutter/issues/19434
+      await Future<void>.delayed(const Duration(milliseconds: 250));
+
+      await driver.forceGC();
```

Comment:
IMO I would not do either of these. the jank is real, it should be in the benchmarks.